packages/measured-signalfx-reporter/lib/registries/SignalFxSelfReportingMetricsRegistry.js

```diff
@@ -0,0 +1,48 @@
+const { NoOpMeter, Timer } = require('measured-core');
+const { SelfReportingMetricsRegistry } = require('measured-reporting');
+const { validateTimerOptions } = require('measured-reporting').inputValidators;
+
+/**
+ * A SignalFx Self Reporting Metrics Registry that disallows the use of meters as they waste DPM and accomplished using a counter.
```

Comment:
nitpick: I'd maybe write this as:

A SignalFx Self Reporting Metrics Registry that disallows the use of meters.  Meters don't make sense to use with SignalFx because the rate aggregations can be done within SignalFx itself.  Meters simply waste DPM (Datapoints per Minute).

Reply:
- [ ] Update language here.